docs/deployment/feature_gates.md

```diff
@@ -26,6 +26,7 @@ The following tables are a summary of the feature gates that you can set on diff
 | ManagedIstio | `false` | `Alpha` | `1.5` | |
 | APIServerSNI | `false` | `Alpha` | `1.7` | |
 | MountHostCADirectories | `false` | `Alpha` | `1.11.0` | |
+| SeedChange | `false` | `Alpha` | `1.12.0` | |
```

Comment:
Can you add it to the `## List of feature gates`, too?

Reply:
It is there already ...